### ISSUE TITLE
Image display problem

### ISSUE THREAD

**Issue by dlogan** [CellProfiler/CellProfiler-Analyst#47]
Below is the same image, as displayed with v11710 (Left) and CPA compiled 2014_04_01 (Right):
![screen shot 2014-05-12 at 4 31 12 pm](https://cloud.githubusercontent.com/assets/1225595/2950779/a6813fe2-da1e-11e3-837b-41f9e1638bad.png)

This wonkiness is the same for all images for the newer CPA version, at least for this properties file, which is from Rakesh's fibroblast screen (MetaXpress 16-bit tiffs -- nothing unusual):
\\\iodine\imaging_analysis\2009_01_20_iPSneuronalcells_SchreiberLab\2013_06_26_Missing_DMSO\2009_01_20_iPsNeuronal_RakeshKarmacharya_Expt_2013_02_19_MissingDMSO_INSPECTING_POSSIBLE_HITS.properties

**Comment by dlogan** [comment]
This display issue occurred on one of my Windows VMs (David_8GB) but looked fine on another (Malaria).  I get this error 
```
Traceback (most recent call last):
  File "imagereader.pyc", line 15, in __init__
  File "bioformats\__init__.pyc", line 20, in <module>
  File "cellprofiler\utilities\jutil.pyc", line 163, in <module>
  File "cellprofiler\utilities\javabridge.pyc", line 12, in <module>
  File "cellprofiler\utilities\javabridge.pyc", line 10, in __load
ImportError: DLL load failed: The specified module could not be found.

ImageReader failed to import load_using_bioformats from CellProfiler, will fall back on PIL and TiffFile.
```

Hmm, on the VM CPA that does work properly, I get this warning:
```
Loading image from "c:\users\develo~1\appdata\local\temp\tmpdnattc.tif"
Loading image from "c:\users\develo~1\appdata\local\temp\tmp562xec.tif"
Loading image from "c:\users\develo~1\appdata\local\temp\tmpbb77h0.png"
Loading image from "c:\users\develo~1\appdata\local\temp\tmpxnmrpo.png"
WARNING: CPA found 3 channels in the "ParasiteArea" image at /imaging/analysis/2010_05_18_Malaria_Liver_Infection_Bhatia_Lab/Analysis/2015_03_02_PV_20x/52589/e11/96wp-pv-20x_e11_s20_w2b98c870f-8c32-4a09-b559-48904a14e0be_ParasiteArea.png, but it will only load the first 1 as specified by properties field channels_per_image.
```

So maybe it is an issue with a saved image that is color (even though the pipeline saves the colormap as "gray")?

**Comment by LeeKamentsky** [comment]
I think that the first error happened because the old bioformats directory in CellProfiler is still there. CellProfiler's source is probably on the PYTHONPATH, so it's picking up the wrong bioformats from there. PYTHONPATH shouldn't be in the default environment for the VMs - if it is, it should be removed. Also, if the CellProfiler source is the head of the master branch, the bioformats directory and jutil.pyc and javabridge.pyc should be deleted.

The second problem, I'm not sure, I forget whether PNGs are always color or not. I would ignore the warning.

**Comment by dlogan** [comment]
Hmm, I removed PYTHONPATH from a Windows VM and it didn't solve the display issue (I logged off and back on, too).  And to be clear, I am using the compiled CPA - would it pick up the bioformats directory, etc?

However, I now suspect updating Java is an issue.  Could that make sense?  I updated Java to version 8 on one VM and it shows the display issue.  The other VM still has Java Version 6 Update 31 and it does not display this issue.  This is quite annoying.

**Comment by LeeKamentsky** [comment]
I think it's more likely that you have jutil.pyc still in cellprofiler/utilities

**Comment by AnneCarpenter** [comment]
I"m guessing this resolved itself w java-related changes over time. Please re-open if that's not the case!
